o4/o4.py

```diff
@@ -1092,13 +1092,43 @@ def move_except(from_dir, to_dir, but_not):
     check_call(cmd)
     if not discard:
         savedir = source.replace('cleaning', time.strftime('cleaned'))  # @%Y-%m-%d,%H:%M'))
+        if os.path.exists(savedir):
+            shutil.rmtree(savedir)
```

Comment:
This is dangerous, they will lose data. I'd rather we just moved it to `.o4/1-cleaned` or something like it

Reply:
Am I the only one that has a panic attack before testing anything with rmtree/`rm -fr` in it?

Reply:
I test it with a print first

Reply:
And still I basically need new underwear after I tried it for realz